src/Tickets/Seating/Frontend/Session.php

```diff
@@ -358,11 +358,6 @@ public function confirm_all_reservations( bool $delete_token_session = true ): b
 	 * @return array|null The reservations for the ticket and post.
 	 */
 	public function get_post_ticket_reservations( int $post_id = null, int $ticket_id = null ): ?array {
-		// Bail while in admin side always. There are no reservations in the admin.
-		if ( is_admin() ) {
-			return null;
-		}
-
```

Comment:
This method that is just checking if the reservation data exists. No need to keep it behind admin check.